target/linux/rockchip/patches-6.12/403-01-v6.13-arm64-dts-rockchip-add-and-enable-gpu-node-for-Radxa.patch

```diff
@@ -0,0 +1,25 @@
+From a98053d098c4ad91a45a3a55604d9574dfc6ffdb Mon Sep 17 00:00:00 2001
```

Comment:
please rename it to start from `0xx` as it's a backport, the same below.

Reply:
All backported patches have been renumbered.